### PR TITLE
Fix NameError

### PR DESCRIPTION
**Pull request by tipabu**
See #826

### CODE REVIEW

**Review by itamarst — APPROVED** [review]
Oops.

File: tests/wsgi_test.py [hunk 1]
```diff
@@ -1939,12 +1939,31 @@ def test_rfc9112_reject_bad_request(self):
...
+            b'Content-Length: 0\r\n'
             b'Host: localhost\r\n'
             b'\r\n')
         result = read_http(sock)
         self.assertRaises(ConnectionClosed, read_http, sock)
 
+    def test_content_length_and_transfer_encoding_escape_hatch(self):
+        class LegacyClientProtocol(wsgi.HttpProtocol):
+            def __init__(self, *args, **kwargs):
+                kwargs['reject_bad_requests'] = False
```
[the remark this answers is not in the document]
Comment (itamarst) [reply tests/wsgi_test.py:1951]: That seems fine, do you want to do it in this PR?

[the remark this answers is not in the document]
Comment (tipabu) [reply tests/wsgi_test.py:1951]: Sure.

**Comment by jayofdoom** [comment]
@itamarst @4383 Can we get this in an eventlet release to unbreak OpenStack Swift?

**Comment by 4383** [comment]
@jayofdoom: Preparing => https://github.com/eventlet/eventlet/issues/897

**Pull request merged**